includes/functions/functions_email.php

```diff
@@ -392,7 +392,7 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
       } catch (Exception $e) {
       }
       if (!$success) {
-        $msg = sprintf(EMAIL_SEND_FAILED . '&nbsp;'. $mail->ErrorInfo, $to_name, $to_email_address, $email_subject);
+        $msg = sprintf(EMAIL_SEND_FAILED . '&nbsp;'. str_replace('%', '%%',$mail->ErrorInfo), $to_name, $to_email_address, $email_subject);
```

Comment:
Would it be better to refactor `$mail->ErrorInfo` out of the sprintf?

Maybe 

```
$msg = sprintf(EMAIL_SEND_FAILED, $to_name, $to_email_address, $email_subject) .  '&nbsp;'. $mail->ErrorInfo; 
```

Reply:
To quote Homer Simpson DOH!!!
That's why we have reviews. Much better and more efficient.